package.json

```diff
@@ -52,5 +52,5 @@
     "url": "https://github.com/ubilabs/google-maps-react-hooks/issues"
   },
   "author": "",
```

Comment:
Didn't you remove this already? 🤔 
```suggestion
```

Reply:
Only for the examples `package.json`. 

We wanted to add an author here, but I am not sure yet what exactly we want to put 👍 So I keep it as a reminder ✨